src/TenantCloud/APIVersioning/Constrain/BadConstraintException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace TenantCloud\APIVersioning\Constrain;
```

Comment:
Constrain or constraint? I can see two variants. I guess it would be better to have one naming